docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -442,7 +444,10 @@ the expressions have NULL values, the function returns a NULL value.
 | <<nvl_function,NVL Function>>                                   | Returns the value of the first operand unless it is NULL, in which case it returns the value of the second operand.
 | <<user_function,USER Function>>                                 | Returns either the database user name of the current user who invoked the function or the database user name
 associated with the specified user ID number.
-| <<sys_guid_function,SYS_GUID Function>>                         | Returns a globally unique identifier.
+| <<sys_guid_function,SYS_GUID Function>>                         | Returns a globally unique identifier. (Oracle extention)
+| <<uuid_function,UUID Function>>                                 | Returns a globally unique identifier.
```

Comment:
How does this contrast to SYS_GUID? Is UUID standard? Or is it like some other vendor?

Reply:
UUID is MySQL function. I don't think ANSI define standard of SQL functions.  In SQL Server, it is called newId(). So let me add comments like SYS_GUID, by adding "similar to MySQL UUID function"?

Reply:
That's fine.